job-service-db/src/main/resources/procedures/dropTaskTables2.sql

```diff
@@ -0,0 +1,62 @@
+--
+-- Copyright 2016-2020 Micro Focus or one of its affiliates.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+/*
+ *  Name: internal_drop_task_tables2
+ *
+ *  Description:
+ *  Drops all task tables belonging to the specified task and all its subtasks
+ *  This is a procedure that commits on bulks whose size is defined by the maxBeforeCommit variable.
```

Comment:
Is the `maxBeforeCommit` variable meant to be configurable?  Can it's size be automatically chosen based on `max_locks_per_transaction`?